app/lib/get-workflows-in-order.coffee

```diff
@@ -1,5 +1,7 @@
-getWorkflowsInOrder = (project, query) ->
+getWorkflowsInOrder = (project, query = {}) ->
   order = project.configuration?.workflow_order ? []
+  unless query?.page_size?
```

Comment:
Not a blocker for me but can't you just set this query param where you need to call it ala [here](https://github.com/zooniverse/Panoptes-Front-End/blob/1f94f898f3480f1659aa47604d3af2196eb26a9b/app/pages/project/home.cjsx#L24)